MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.storyboard

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="16096" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="15705" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
```

Comment:
Are those storyboard changes expected?

Reply:
I assume yes. Currently we add the prompt options directly in the storyboard instead of adding them programatically. 
Regarding the toolsVersion, i think it is because my XCode (10) is lower than the previous file modifier. I think it should not be a problem for those who uses newer version of XCode :)